### ISSUE TITLE
Crash in BugsnagSystemState.m:224

### ISSUE THREAD

**Issue by sethfri** [bugsnag/bugsnag-cocoa#898]
### Describe the bug
Crash of unknown origin coming from Bugsnag. We have seen 1 crash coming from 1 user so far.

### Steps to reproduce
Unknown

### Environment
* Bugsnag version: 6.2.3
* CocoaPods version: 1.9.3
* Carthage version (if any): n/a
* iOS/tvOS/macOS version(s): 14.0.0
* Simulator or physical device: iPhone 7 Plus
* XCode version: 12.1
* Swift version (if applicable): 5.1

### Stack Trace

<details><summary>Stack trace:</summary>

```
SIGSEGV 0x00000009edc56620

Crashed: com.apple.root.default-qos
0  libobjc.A.dylib                0x1c484cca0 objc_retain + 16
1  MyApp                           0x104c818a8 -[BugsnagSystemState sync] + 224 (BugsnagSystemState.m:224)
2  libdispatch.dylib              0x1b0c41298 _dispatch_call_block_and_release + 24
3  libdispatch.dylib              0x1b0c42280 _dispatch_client_callout + 16
4  libdispatch.dylib              0x1b0be7224 _dispatch_queue_override_invoke + 692
5  libdispatch.dylib              0x1b0bf411c _dispatch_root_queue_drain + 376
6  libdispatch.dylib              0x1b0bf48e4 _dispatch_worker_thread2 + 124
7  libsystem_pthread.dylib        0x1f68ec568 _pthread_wqthread + 212
8  libsystem_pthread.dylib        0x1f68ef874 start_wqthread + 8
```
</details>

**Comment by sethfri** [comment]
I'm assuming this is a dupe of #886, but just wanted to double check since it's coming from a different line in that file

**Comment by nickdowell** [comment]
Hi @sethfri you are correct, this is a duplicate of #886 - the line number changed because of some changes to that file between 6.2.1 and 6.2.3

**Comment by johnkiely1** [comment]
Hi @sethfri 
Closing this as it is a duplicate and fix was released in v6.2.5
